.circleci/config.yml

```diff
@@ -195,27 +187,11 @@ jobs:
     steps:
       - checkout
       - setup_remote_docker
-      - run:
-          name: Install modules
-          command: |
-            pip install awscli --upgrade
-      - run:
-          name: Configure AWS Access Key ID
-          command: |
-            aws configure set aws_access_key_id \
-            $AWS_ACCESS_KEY_ID \
-            --profile default
-      - run:
-          name: Configure AWS Secret Access Key
-          command: |
-            aws configure set aws_secret_access_key \
-            $AWS_SECRET_ACCESS_KEY \
-            --profile default
-      - run:
-          name: Configure AWS default region
-          command: |
-            aws configure set region $AWS_REGION \
-            --profile default
+      - aws-cli/setup:
+          role-arn: "arn:aws:iam::$ORG_AWS_ACCOUNT_ID:role/CircleCI-OIDC-Connect"
+          profile-name: "OIDC-Profile"
+          role-session-name: "frontend-deploy"
+          session-duration: "1800"
```

Comment:
Session duration is set to 1800 on frontend and backend deploy jobs. Will this be enough? Since we create a db snapshot during backend deployment.

Reply:
Thank you for the review, @Aadesh-Baral . I see in past runs that backend deploy takes less than 30min. But I have  increased the duration to 45min anyway to include a buffer.